Firebase/Database/Api/FIRServerValue_unreleased.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRServerValue_unreleased_h
+#define FIRServerValue_unreleased_h
+
+#import "FIRServerValue.h"
+
+@interface FIRServerValue (Unreleased)
+
+/**
+ * Adds the given delta to the ref's current value.
```

Comment:
Nit: Please replace usage of "ref's" with something else. Example would be: 

```
Adds the given delta to the current value at the current location.
```

We should probably not use "current" twice, but it is too early for me to look at a thesaurus.